<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messageWindow</class>
 <widget class="QWidget" name="messageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 15px;
background-color: rgba(61, 255, 181,0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="20,80" columnstretch="90,10">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="logoutGroupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(76, 165, 34);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
border-top-right-radius: 0px;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="logoutGroupBoxLayout" columnstretch="30,40,20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <spacer name="horizontalSpacerLogout">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>28</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sistema</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="closeDialogButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(76, 165, 34);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 15px;
}
QPushButton:hover{
	
	background-color: rgb(190, 67, 42);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="logoutOptionsGroupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius: 0px;
border-top-right-radius: 0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="20,10,60,20">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>35</number>
      </property>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="acceptDialogButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(116, 116, 116);
border-radius: 15px;
}
QPushButton:hover{
	
	background-color: rgb(76, 165, 34);
}</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="messageLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>messageWindow</receiver>
   <slot>closeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>messageWindow</receiver>
   <slot>closeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeDialog()</slot>
 </slots>
</ui>
